docs/docs/behaviors/turbo.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Turbo Behavior
+sidebar_label: Turbo Key
+---
+
+## Summary
+
+The turbo behavior will repeatedly trigger a behavior after a specified amount of time.
```

Comment:
I am a bit unclear on how exactly this works and it would be good to clarify further here. I think by default, (no toggle term) it only repeats while the key is held, and cancels repeating when it is released, is it?

But if toggle term is set and tap duration is shorter than that, it stays active and the next press cancels it?